vuepress/docs/next/docs/compose/local-hub-overview.md

```diff
@@ -44,9 +40,9 @@ responsible for the low-level communication with the K8s cluster API.
 
 ## Architecture
 
-![ECR Architecture](./img/ecr-architecture.png)
+![Local Hub Architecture](./img/local-hub-architecture.png)
 
-From an architectural point of view, the ECR is composed of: 
+From an architectural point of view, the Local Hub is composed of: 
 1. The `EntandoDeBundles` which contain the metadata associated with a bundle
```

Comment:
Add to #1 or add a new line. "The list of EntandoDeBundles (or Entando Bundles) is displayed as the "Local Hub" in the App Builder."